physics/GFS_MP_generic_post.F90

```diff
@@ -101,6 +109,10 @@ subroutine GFS_MP_generic_post_run(
       real(kind=kind_phys) :: crain, csnow, onebg, tem, total_precip, tem1, tem2, ttend
       real(kind=kind_phys), dimension(im) :: domr, domzr, domip, doms, t850, work1
 
+      real :: snowrat,grauprat,icerat,curat,prcpncfr,prcpcufr
+      real :: rhonewsnow,rhoprcpice,rhonewgr,rhonewice
+      real(kind=kind_phys), parameter :: rhowater = 1000.0_kind_phys
```

Comment:
This constant is already defined in the host (see rhowater in physcons.F90). You should be able to pass it in via the argument list by using the following metadata:
`[rhowater]`
  `standard_name = fresh_liquid_water_density_at_0c`
  `long_name = density of liquid water`
  `units = kg m-3`
   `dimensions = ()`
    `type = real`
    `kind = kind_phys`
    `intent = in`